packages/core/useTransition/index.ts

```diff
@@ -193,14 +208,14 @@ export function executeTransition<T extends number | number[]>(
   })
 }
 
-// option 1: reactive number
-export function useTransition(source: MaybeRefOrGetter<number>, options?: UseTransitionOptions): ComputedRef<number>
```

Comment:
The first option was simplified and moved to the end to allow for transitioning unknown types.